tests/cases/compiler/referenceSatisfiesExpression.ts

```diff
@@ -0,0 +1,33 @@
+// checkReferenceExpression -> skip OuterExpressionKinds.SatisfiesExpressions
```

Comment:
I'd probably drop those comments here (in this whole file) - they refer to implementations details that might change in the future